numpyro/contrib/module.py

```diff
@@ -18,7 +18,7 @@
 ]
 
 
-def flax_module(name, nn_module, *, input_shape=None):
+def flax_module(name, nn_module, *args, input_shape=None, **kwargs):
```

Comment:
The reason we have `*` here is to avoid breaking changes for whatever reason in the future. I guess having `*args` is not important. If a user uses an arg here and we change the way it works in the future, things might break. Also, can you also add docstring for `kwargs`?

Reply:
👍 sounds good.